.github/workflows/test_and_build.yml

```diff
@@ -151,6 +151,17 @@ jobs:
       - run: ./scripts/install_zig.sh
       - run: zig/zig build fuzz_vsr_superblock_quorums -- --seed 123
 
+  # This both checks that the hash_log builds and acts as a regression test for 
+  # https://github.com/tigerbeetledb/tigerbeetle/issues/404
+  fuzz_hash_log:
```

Comment:
Can you add this to the `bors_tigerbeetle` job?

Reply:
I see, bors doesn't run all jobs?